mix.exs

```diff
@@ -30,9 +30,9 @@ defmodule Scholar.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.29", only: :docs},
-      {:nx, github: "elixir-nx/nx", sparse: "nx", override: true},
+      {:nx, "~> 0.6", override: true},
       {:nimble_options, "~> 0.5.2 or ~> 1.0"},
-      {:exla, github: "elixir-nx/exla", sparse: "exla", only: [:test, :dev]},
+      {:exla, github: "elixir-nx/exla", sparse: "exla", only: [:test, :dev], override: true},
```

Comment:
I don't if this line should stay like this

Reply:
We should depend on it as regular Hex package! Any reason not to?

Reply:
Ok, I needed to update `xla` in `mix.lock`. A also bumped ex_doc to 0.30.